doc/tech/glossary.md

```diff
@@ -146,15 +146,24 @@ Terms not defined by Kubernetes:
 - Observability = Observability is a measure of how well internal states of a system can be inferred from knowledge of its external outputs. ([Wikipedia:Observability](https://en.wikipedia.org/wiki/Observability))
 
 <a name="1.7"></a>
-## 1.7 Other Referenced Terminology
+## 1.7 Test Related Terminology
+
```

Comment:
New intro
```suggestion
The main objective of CNTT performance testing is to understand if a potential BOM is functioning correctly to get expected performance. It requires a set of performance testing tools (open source) that help with the dimensioning of a solution by an operator. Testing results may be useful to compare infrastructure capabilites between the System Under Test (SUT) and a CNTT reference implementation of RA-2. Performance testing for the purpose of comparing between different commercial implementations is not a goal of CNTT. Performance testing relies on well established benchmark specifications.
```

Reply:
BOM -- acronym first use. If it is Bill of Materials then we can't say that "a potential BOM is functioning correctly..."

Reply:
Also, why does this require an operator? Could performance testing be conducted on a reference implementation stood up by a testing organisation?

Reply:
This PR is for text in the glossary and to keep consistent I have not added an introduction. Most of the text is worked into the definitions now.

BOM and specific ref to operators removed.

Reply:
Removed BOM and specific ref to operators

Reply:
@trevgc @acm This Intro change has not been made